npu.user.js

```diff
@@ -606,8 +606,12 @@
         }
         var keepAlive = function() {
           window.setTimeout(function() {
+              var pages = ["main.aspx", "main.aspx?ismenuclick=true&ctrl=inbox", "main.aspx?ctrl=0203&ismenuclick=true", "main.aspx?ismenuclick=true&ctrl=outbox"];
+              var index = Math.floor(Math.random() * Math.floor(pages.length));
```

Comment:
`pages.length` is an integer anyway, there’s no need to round it, just use it as is:
```javascript
Math.floor(Math.random() * pages.length)
```